src/shared/components/ShareWithColleagues/ShareWithColleagues.tsx

```diff
@@ -108,36 +147,50 @@ const ShareWithColleagues: FC<ShareWithColleaguesProps & UserProps> = ({
 						return (
 							<li key={index} className="c-colleague-info-row">
 								<div className="c-colleague-info-row__avatar">
-									<Avatar
-										initials={
-											contributorUser.firstName && contributorUser.lastName
-												? contributorUser.firstName[0] +
-												  contributorUser.lastName[0]
-												: contributorUser.email.slice(0, 2).toUpperCase()
-										}
-										image={contributorUser.profileImage}
-									/>
+									{contributor.profileId ? (
+										<Avatar
+											initials={
+												contributor.firstName && contributor.lastName
+													? (
+															contributor.firstName[0] +
+															contributor.lastName[0]
+													  ).toLocaleUpperCase()
+													: contributor.email?.slice(0, 2).toUpperCase()
+											}
+											image={contributor.profileImage}
+										/>
+									) : (
+										<div className="c-colleague-info-row__avatar--pending" />
```

Comment:
empty row for pending? or is the email address for the invited user added later?

Reply:
The empty div is a circle with a dashed border to replace the non existing avatar